CreateGroup/index.ts

```diff
@@ -0,0 +1,18 @@
+import { AzureFunction, Context, HttpRequest } from '@azure/functions';
+import { GroupModel} from '../shared/groupUser/group.model';
+import getConnection from '../shared/mogoose';
+
+
+const createGroup: AzureFunction = async (
+    context: Context,

```

Comment:
We add this underscore when we don't use a parameter in the function
```suggestion
    _context: Context,
```